pkg/types/nutanix/helpers.go

```diff
@@ -150,7 +167,7 @@ func WaitForTask(clientV3 nutanixclientv3.Service, taskUUID string) error {
 		time.Sleep(sleepTime)
 	}
 	if !finished {
-		return errors.Errorf("timeout while waiting for task UUID: %s", taskUUID)
+		return errors.Errorf("timeout while waiting for task UUID: %s, used_time: %s", taskUUID, time.Since(start))
```

Comment:
```suggestion
		return fmt.Errorf("timeout while waiting for task UUID: %s, used_time: %s", taskUUID, time.Since(start))
```

Reply:
@barbacbd Since this PR is manually cherrypick-ed from the PRs https://github.com/openshift/installer/pull/9406 (for release-4.17), https://github.com/openshift/installer/pull/9399 (for release-4.18) and https://github.com/openshift/installer/pull/9377 (for master branch), and using errors.Errorf() or fmt.Errorf() basically is the same effect.